site/src/components/WorkspaceSchedule/WorkspaceSchedule.tsx

```diff
@@ -4,8 +4,10 @@ import Typography from "@material-ui/core/Typography"
 import ScheduleIcon from "@material-ui/icons/Schedule"
 import cronstrue from "cronstrue"
 import dayjs from "dayjs"
+import advancedFormat from "dayjs/plugin/advancedFormat"
 import duration from "dayjs/plugin/duration"
 import relativeTime from "dayjs/plugin/relativeTime"
+import timezone from "dayjs/plugin/timezone"
```

Comment:
**Review**:

Both are needed to manipulate time in the user's timezone and display timezone information (specifically where you see `.format("z")`